src/detail/details.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import request from 'superagent';
+import Spinner from '../search/spinner.js'
+export default class DetailPage extends Component {
+    state = {
+        pokemonData: {},
+    }
+
+    componentDidMount = async () => {
+        this.setState({ loading: true })
+
+        const data = await request.get(`https://pokedex-alchemy.herokuapp.com/api/pokedex?pokemon=${this.props.match.params.pokemonName}`);
```

Comment:
Since we are getting just one pokemon back, you won't need to use the .find on line 16. :)